app/experimenter/nimbus-ui/src/components/FormOverview/index.tsx

```diff
@@ -123,34 +127,37 @@ const FormOverview = ({
         <Form.Control
           {...nameValidated("hypothesis")}
           as="textarea"
-          rows={3}
-          defaultValue={experiment?.hypothesis || ""}
+          rows={5}
+          defaultValue={experiment?.hypothesis || hypothesisDefault || ""}
         />
         <Form.Text className="text-muted">
           You can add any supporting documents here.
         </Form.Text>
         <FormErrors name="hypothesis" />
       </Form.Group>
 
-      <Form.Group controlId="application">
-        <Form.Label>Application</Form.Label>
-        <Form.Control
-          {...nameValidated("application")}
-          as="select"
-          defaultValue={experiment?.application || ""}
-        >
-          <option value="">Select...</option>
-          {application!.map((app, idx) => (
-            <option key={`application-${idx}`} value={app!.value as string}>
-              {app!.label}
-            </option>
-          ))}
-        </Form.Control>
-        <Form.Text className="text-muted">
-          Experiments can only target one Application at a time.
-        </Form.Text>
-        <FormErrors name="application" />
-      </Form.Group>
+      {experiment ? (
+        <React.Fragment>
+          <p>Application</p>
+          <p>{experiment.application}</p>
+        </React.Fragment>
+      ) : (
+        <Form.Group controlId="application">
+          <Form.Label>Application</Form.Label>
+          <Form.Control {...nameValidated("application")} as="select">
+            <option value="">Select...</option>
+            {application!.map((app, idx) => (
+              <option key={`application-${idx}`} value={app!.value as string}>
+                {app!.label}
+              </option>
+            ))}
+          </Form.Control>
+          <Form.Text className="text-muted">
+            Experiments can only target one Application at a time.
+          </Form.Text>
+          <FormErrors name="application" />
+        </Form.Group>
+      )}
```

Comment:
Actually this whole experiment check could probably be moved further in so we can keep the visual consistency of the form label and grouping all intact. Something like this:

```tsx
<Form.Group controlId="application">
  <Form.Label>Application</Form.Label>
  {experiment ? (
    <p>
      <b>
        {
          application!.find((a) => a?.value === experiment.application)
            ?.label
        }
      </b>
    </p>
  ) : (
    <>
      <Form.Control {...nameValidated("application")} as="select">
        <option value="">Select...</option>
        {application!.map((app, idx) => (
          <option key={`application-${idx}`} value={app!.value as string}>
            {app!.label}
          </option>
        ))}
      </Form.Control>
      <Form.Text className="text-muted">
        Experiments can only target one Application at a time.
      </Form.Text>
      <FormErrors name="application" />
    </>
  )}
</Form.Group>
```

Reply:
Good call 👍